src/AWS.Deploy.Recipes/CdkTemplates/ConsoleAppECSFargateService/Generated/Recipe.cs

```diff
@@ -0,0 +1,235 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Amazon.CDK;
+using Amazon.CDK.AWS.EC2;
+using Amazon.CDK.AWS.ECS;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.ECR;
+
+using AWS.Deploy.Recipes.CDK.Common;
+
+using ConsoleAppEcsFargateService.Configurations;
+using Amazon.CDK.AWS.ApplicationAutoScaling;
+
+// This is a generated file from the original deployment recipe. It is recommended to not modify this file in order
+// to allow easy updates to the file when the original recipe that this project was created from has updates.
+// To customize the CDK constructs created in this file you should use the AppStack.CustomizeCDKProps() method.
+
+namespace ConsoleAppEcsFargateService
+{
+    using static AWS.Deploy.Recipes.CDK.Common.CDKRecipeCustomizer<Recipe>;
+
+    public class Recipe : Construct
+    {
+        public IRole? AppIAMTaskRole { get; set; }
+
+        public IVpc? AppVpc { get; set; }
+
+        public ICluster? EcsCluster { get; set; }
+
+        public ContainerDefinitionOptions? AppContainerDefinition { get; set; }
+
+        public AwsLogDriver? AppLogging { get; set; }
+
+        public FargateTaskDefinition? AppTaskDefinition { get; set; }
+
+        public FargateService? AppFargateService { get; set; }
+
+        public ScalableTaskCount? AutoScalingConfiguration { get; private set; }
+
+        public string AutoScaleTypeCpuType { get; } = "AutoScaleTypeCpuType";
+        public string AutoScaleTypeMemoryType { get; } = "AutoScaleTypeMemoryType";
+
+
+        public Recipe(Construct scope, IRecipeProps<Configuration> props)
+            : base(scope, "Recipe")
```

Comment:
There should be a comment explaining why the construct id is constant here. And changing it will impact the stack.

Reply:
Done